code/_onclick/cyborg.dm

```diff
@@ -15,6 +15,8 @@
 		return
 	changeNext_click(1)
 
+	if(is_ventcrawling(src)) // To stop drones interacting with anything while ventcrawling
```

Comment:
This also stops them from examining stuff and what not. Which is fine but it doesn't sound like a fix but more like a tweak

Reply:
Just tested it on master, and it looks like they can't examine anyway.